community/serializers.py

```
@@ -309,6 +309,35 @@ def to_representation(self, instance):
+    )
+
+    class Meta:
+        model = Comment
+        fields = [
+            "id",
+            "content",
+            "postId",
+            "postTitle",
+            "boardType",
+            "createdAt",
+        ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)